src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
 
-const ext = joinpath(@__DIR__, "..", "deps", "ext.jl")
-isfile(ext) || error("Unable to load $ext\n\nPlease run Pkg.build(\"CUDAdrv\") and restart Julia.")
-include(ext)
+const ext = joinpath(dirname(@__DIR__), "deps", "ext.jl")
+if isfile(ext)
+    include(ext)
+else
+    error("Unable to load dependency file $ext.\nPlease run Pkg.build(\"CUDAdrv\") and restart Julia.")
+end
 const libcuda = libcuda_path
 
-include("util/logging.jl")
+include(joinpath("util", "logging.jl"))
```

Comment:
Are these change really necessary under Windows? I understand the intention behind them, but I find it a bit odd.